### PR TITLE
Fix executable args for phpdbg

### LINKED ISSUE #571: sh: exec: line 1: /usr/local/bin/phpdbg -qrr: not found
| Q | A
| --- | ---
| ParaTest version | 6.1.1
| PHPUnit version | 9.5.0
| PHP version | 7.4.3

#### Summary

Paratest generates incorrect command when using phpdbg for code coverage.

#### Current behavior

Running `phpdbg -qrr vendor/bin/paratest -p12` throws an exception:

```
In WorkerCrashedException.php line 36:
                                                                                                                                                                                                          
  The command "'/usr/local/bin/phpdbg -qrr' '/src/vendor/phpunit/phpunit/phpunit' '--configuration' '/src/phpunit.xml' '--no-logging' '--no-coverage' '--printer' 'ParaTest\Runners\PHPUnit\Worker\NullPhpunitPrinter' '--log-junit' '/tmp/PT_HkPaNg' '/src/tests/Unit/Casts/TimestampTest.php'" failed.                                                                                                        
                                                                                                                                                                                                          
  Exit Code: 127(Command not found)                                                                                                                                                                       
                                                                                                                                                                                                          
  Working directory: /src                                                                                                                                                                                 
                                                                                                                                                                                                          
  Output:                                                                                                                                                                                                 
  ================                                                                                                                                                                                        
                                                                                                                                                                                                          
                                                                                                                                                                                                          
  Error Output:                                                                                                                                                                                           
  ================                                                                                                                                                                                        
  sh: exec: line 1: /usr/local/bin/phpdbg -qrr: not found 
```

I guess that `'/usr/local/bin/phpdbg -qrr'` should be exploded into separate parts.
Reproduced on Ubuntu 20.04.1 LTS and Alpine 3.11.3.

#### How to reproduce: command, code and error stack trace

Run `phpdbg -qrr vendor/bin/paratest -p12`

#### Expected behavior

Paratest should generate
```
"'/usr/local/bin/phpdbg' '-qrr' '/src/vendor/phpunit/phpunit/phpunit' '--configuration' '/src/phpunit.xml' '--no-logging' '--no-coverage' '--printer' 'ParaTest\Runners\PHPUnit\Worker\NullPhpunitPrinter' '--log-junit' '/tmp/PT_HkPaNg' '/src/tests/Unit/Casts/TimestampTest.php'"
````

### PR DESCRIPTION
**Pull request by Arthur-Sk**
Closes #571

### CODE REVIEW

File: src/Runners/PHPUnit/Worker/RunnerWorker.php [hunk 1]
```diff
@@ -31,7 +32,12 @@ public function __construct(ExecutableTest $executableTest, Options $options, in
     {
         $this->executableTest = $executableTest;
 
-        $args = [(new PhpExecutableFinder())->find()];
+        $args       = [];
+        $executable = (new PhpExecutableFinder())->find();
+        if ($executable !== false) {
+            $args = explode(' ', $executable);
+        }
+
```
Comment (Slamdunk) [inline src/Runners/PHPUnit/Worker/RunnerWorker.php:40]: I'd prefer:

```php
$phpFinder = new PhpExecutableFinder();
$args = [$phpFinder->find(false)];
$args = array_merge($args, $phpFinder->findArguments());
```

May you apply my suggestion and test it locally please?

**Review by Slamdunk — CHANGES_REQUESTED** [review]

**Review by Slamdunk — CHANGES_REQUESTED** [review]
May you apply the fix also to `\ParaTest\Runners\PHPUnit\Worker\WrapperWorker` please?

**Comment by Arthur-Sk** [comment]
> May you apply the fix also to `\ParaTest\Runners\PHPUnit\Worker\WrapperWorker` please?

Sure

**Review by Slamdunk — APPROVED** [review]

**Comment by Slamdunk** [comment]
Thank you :+1:

**Pull request merged**